renderer/components/test-results/TestResultsContainer.js

```diff
@@ -107,21 +107,26 @@ const groupRowsByMonth = (rows) => {
     return []
   }
 
-  // We assume the rows are sorted from newest to oldest
+  // We assume the rows are sorted from oldest to newest
   const start = moment(rows[0].start_time)
   const end = moment()
   let range = moment.range(start, end).snapTo('month')
   let byMonth = {}
   Array.from(range.by('month', { excludeEnd: false})).map(m => {
     byMonth[m.format('YYYY-MM-01')] = []
   })
+
   // XXX I think there is a better way to do this without so many list
   // reversals.
   rows.reverse().map(row => {
     const month = moment(row.start_time).format('YYYY-MM-01')
     byMonth[month].push(row)
   })
-  return Object.keys(byMonth).sort().reverse().map(key => [key, byMonth[key]])
+  return Object.keys(byMonth)
+    .filter(month => byMonth[month].length > 0 )
```

Comment:
Is this to avoid the empty section in the result list? Might be useful to add the reason here.

Reply:
Done.